packages/types/src/transaction.ts

```diff
@@ -1,7 +1,10 @@
+import type { IncomingMessage } from 'http';
```

Comment:
It occurred to me it might cause problems, but my thinking was that types are only relevant in dev (ours or our users'), and dev always happens in a node context. (I also just tried building a browser app which uses tracing but doesn't have node types installed, and it seemed to build just fine.

That said, you're right that an import-from-nowhere solution is foolproof, so I'm fine doing it that way.